.github/workflows/ci.yaml

```
@@ -0,0 +1,16 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up registry credentials
+      if: ${{ github.event_name != 'pull_request' }}
```

This will on feature branches in user forks, so we should gate the repository_owner too

```suggestion
      if: ${{ github.repository_owner == "ManageIQ" && github.event_name != 'pull_request' }}
```

---

ok - btw, if we only run this on pushes, do we still want `on: [push, pull_request]` or the simpler `on: [push]` ?

---

We want to do the build on both pull requests and pushes so we know a PR doesn't break the build.

The script only does a deploy when the env vars are present, which should only be on push.